packages/extensions-sdk/src/cli/commands/validate.ts

```diff
@@ -0,0 +1,68 @@
+import chalk from 'chalk';
+import ora from 'ora';
+import { log } from '../utils/logger.js';
+
+import validators from './validators/index.js';
+import type { Report } from '../types.js';
+
+type ValidateOptions = {
+	check?: string;
+	verbose?: boolean;
+};
+
+export default async function validate(options: ValidateOptions): Promise<void> {
+	const spinner = ora(chalk.bold('Validating Directus extension...')).start();
+	const reports: Array<Report> = [];
+
+	try {
+		const filteredValidators = validators.filter((value) => {
+			if (options.check) {
+				return options.check.toLowerCase() === value.name.toLowerCase();
+			}
+
+			return true;
+		});
+
+		if (filteredValidators.length < 1) {
+			throw new Error(`No validator selected`);
+		}
+
+		const result = await Promise.allSettled(
+			filteredValidators.map(async (value) => {
+				return await value.handler(spinner, reports);
+			}),
```

Comment:
There is no need to wrap an async function with another async function and await it. This should suffice:

```suggestion
			filteredValidators.map((value) => value.handler(spinner, reports)),
```